app/models/decidim/action_delegator/participant.rb

```diff
@@ -21,7 +21,6 @@ class Participant < ApplicationRecord
       delegate :consultation, to: :setting
       delegate :organization, to: :setting
 
-      validates :setting, presence: true
```

Comment:
Why is this removed?

Reply:
belongs_to :setting already includes presence validation. So, the explicit validation is redundant.

Reply:
great, thanks